### DISCUSSION TITLE
warped_low_imgs = list(warper.warp_images(low_imgs, cameras, camera_aspect)) geive me wrong image

### DISCUSSION THREAD

**Discussion by BALLABLLU** [OpenStitching/stitching#120] [Q&A] [answered]
When I use the 'affine' method, the mask shows the correct rotation, but the image also rotates and is displayed incorrectly. It looks like a kaleidoscope, and the second mask shows what the image is supposed to be. However, the real image shows all the mirrored content(part of the original image) in the black area of the mask.
![4bcc6ed379950f274a66b44f83517e5](https://github.com/OpenStitching/stitching/assets/83478717/ff8e012c-d20c-4199-ae47-b825fa628caf)

**Comment by BALLABLLU** [comment]
yeah, you are right, it works fine but I want to know how to solve
this problem, it did not match perfectly, can I make it more accurate? [image:
4686c1884e54af2e44333eee2f6912b.png]

Lukas Weber ***@***.***> 于2023年8月6日周日 11:55写道：

**Reply by lukasalexanderweber** [reply]
The Image isn't shown

**Comment by BALLABLLU** [comment]
Sorry I will send you

Lukas Weber ***@***.***>于2023年8月6日 周日下午12:59写道：

**Comment by BALLABLLU** [comment]
![unnamed](https://github.com/OpenStitching/stitching/assets/83478717/7d1987e1-9632-4820-b7dd-14d2f182b3bf)

**Reply by lukasalexanderweber** [reply]
This looks fine doesn't it?

**Reply by BALLABLLU** [reply]
![e2ca5cd78f4bbc88fe39181d16e6871](https://github.com/OpenStitching/stitching/assets/83478717/9b965015-0f60-4248-91b8-b0f2be197fe5)
there are doubles where suppose to be a single tree or

**Reply by lukasalexanderweber** [reply, marked as answer]
These are not images where you should use affine mode

**Comment by BALLABLLU** [comment]
I see, thank you!
Lukas Weber ***@***.***>于2023年8月16日 周三下午1:14写道：
